main.tf

```diff
@@ -74,6 +74,31 @@ resource "azapi_resource" "aiserviceconnection" {
   response_export_values = ["*"]
 }
 
+# Azure Machine Learning Compute Instance
+resource "azapi_resource" "computeinstance" {
+  count = var.create_compute_instance ? 1 : 0
+
+  type = "Microsoft.MachineLearningServices/workspaces/computes@2024-04-01"
+  body = jsonencode({
+    properties = {
+      computeType = "ComputeInstance"
+      properties = {
+        enableNodePublicIp = false
+        # enableSSO = bool
+        # idleTimeBeforeShutdown = "string"
```

Comment:
there is a shortcut to comment code `del`

Reply:
Commented code is now removed.